app/assets/javascripts/labels.coffee

```diff
@@ -1,22 +1,24 @@
 ready = ->
   $('.my-tag').select2
-    tags: true
-    multiple: true
-    minimumInputLength: 1
-    createTag: (params) ->
-      term = $.trim(params.term)
-      if term == ''
-        return null
-      {
-        id: term
-        text: term
-        newTag: true
-      }
     width: '100%'
-    tokenSeparators: [
-      ','
-      ' '
-    ]
+    multiple: true
+    maximumSelectionLength: 5
+    closeOnSelect: false
+
+    templateSelection: (data, container) ->
+      $(container).css("background-color", "#"+$(data.element).attr("label_color"));
+      $(container).css("color", "#"+$(data.element).attr("label_font_color"));
+      $(container).children("span").css("color", "#"+$(data.element).attr("label_font_color"));
+
+      $template = $('<span></span>').text(data.text);
+      $template.css({"font-size" : "80%", "font-weight" : "bold", "color" : "#"+$(data.element).attr("label_font_color")});
+      return $template;
+
+    templateResult: (data, container) ->
+      $template = $('<div></div>').text(data.text).addClass("exercise_label");
```

Comment:
Please migrate the css-class as well (move from `exercises.scss` to `tasks.scss` and rename to `task_label`)
Don't forget to search and replace for all occurrences of the old class

Reply:
I moved the content of exercises.scss into tasks.scss and removed exercises.scss.

Reply:
That's not quite what I meant, sorry for being inconclusive.
I wanted you to migrate only the classes you use in this PR (mainly exercise_label/task_label) to the new css-file.
The plan is to slowly migrate all the classes over to the new file, whenever we reimplement a feature and at some point we can just get rid of the old css-file.